src/lib/updates/claimGOOD.js

```diff
@@ -0,0 +1,45 @@
+import API from '../API/api'
+import { fireEvent, GOOD_AIRDROP } from '../analytics/analytics'
+
+const fromDate = new Date('2022/03/19')
+
+const claimGOOD = async (lastUpdate, prevVersion, log, goodWallet, userStorage) => {
```

Comment:
could we just import wallet / storage instances instead of passing them as params ?

Reply:
i'm re-factoring the wallet + userstorage into hooks so that it can be more modular,
this is part of this future change.
you can see my progress at branch `wallet-refactor`

Reply:
ok thanks for clarification